tools/sdk-generation-pipeline/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/oss/mirror/docker.io/library/ubuntu:20.04
+FROM mcr.microsoft.com/dotnet/runtime:3.1-focal
```

Comment:
If you want to get a solid image that doesn't include the dotnet runtime, this [CBL-Mariner image supports node.](https://eng.ms/docs/more/containers-secure-supply-chain/approved-images)

```dockerfile
FROM mcr.microsoft.com/cbl-mariner/base/core:1.0 AS img
RUN tndf install nodejs -y
```

Reply:
dotnet runtime is needed for now, which is used for generating and build dotet sdk.

Reply:
dotnet runtime is needed for now, which is used for generating and build dotet sdk.

Reply:
Awesome, then you're doing exactly the right thing 👍